Assets/_Scripts/Enemy/Enemy.cs

```diff
@@ -40,11 +44,14 @@ void Update()
         if (_isPaused)
             return;
 
-        if (IsLookingDown())
-        {
-            Shoot();
-        }
+        if (IsLookingDown() && _canShoot)
+        {
+           Shoot();
+            
+        }
+        
     }
+    
 
     // TriggerOnAnimationEvent
     public void Shoot()
```

Comment:
The reason the bullets are not facing down when shot is due to the way they're being spawned, they're spawned at the rotation of the enemy, and because the enemy lookingDown() can return anywhere on the angle between 225 and 315 degrees it means the sprite being spawned will be at the same angle. So I think (heavily caveated) a change needs to be made when the projectile is spawned on line 63. enemyProjectileSpawn.rotation needs to be set to a vector of 0, 0, 0, or something along those lines.

Reply:
bullet faces proper way now as I looked into
thank you for showing me the proper area to look